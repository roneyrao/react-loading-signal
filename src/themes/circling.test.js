import React from 'react';
import { shallow } from 'enzyme';
import Circling from './circling';

test('default message', () => {
  const wrapper = shallow(<Circling />);
  expect(wrapper.text()).toBe('loading...');
  expect(wrapper).toMatchSnapshot();
});
test('supplied messages', () => {
  const msgs = (
    <ul>
      <li>msg1</li>
      <li>msg2</li>
    </ul>
  );
  const wrapper = shallow(<Circling message={msgs} />);
  expect(wrapper.find('ul').text()).toBe('msg1msg2');
  expect(wrapper).toMatchSnapshot();
});
