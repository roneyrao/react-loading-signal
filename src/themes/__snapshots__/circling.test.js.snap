// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default message 1`] = `
<div
  className="LoadingSignal__circling"
>
  loading...
</div>
`;

exports[`supplied messages 1`] = `
<div
  className="LoadingSignal__circling"
>
  <ul>
    <li>
      msg1
    </li>
    <li>
      msg2
    </li>
  </ul>
</div>
`;
